chapters/overloaded.tex

```diff
@@ -148,20 +152,22 @@ \section{Overloaded Constructors}\doublelabel{overloaded-constructors}
   be removed}{]}\emph{.}
 \end{itemize}
 
-{[}\emph{By the last restriction the following problem for binary
-operators is avoided: }
+\begin{nonnormative}
+By the last restriction the following problem for binary
+operators is avoided:
 
-\emph{Assume there are two operator record classes C and D that both
+Assume there are two operator record classes C and D that both
 have a constructor from Real. If we want to extend c+c and d+d to
 support mixed operations, one variant would be to define c+d and d+c;
 but then c+2 becomes ambiguous (since it is not clear which instance
 should be converted to). Without mixed operations expressions such as
 c+d are only ambiguous if both conversion from C to D and back from D to
 C are both available, and this possibility is not allowed by the
-restriction above.}{]}
+restriction above.
+\end{nonnormative}
 
-Additionally there is an operator `0' defining the zero-value which can
-also be used to construct an element. The operator `0' for an operator
+Additionally there is an operator `\lstinline!0!' defining the zero-value which can
```

Comment:
It should be \lstinline!'0'!; as the name of the function is '0' with quotes, it seems it was broken before as well.

Reply:
Right.  Fixed (also in _revisions.tex_).